Ginger/Ginger/SolutionWindows/AccessibilityRulePage.xaml.cs

```diff
@@ -0,0 +1,147 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.CoreNET.ActionsLib.UI.Web;
+using Amdocs.Ginger.CoreNET.GeneralLib;
+using Amdocs.Ginger.Repository;
+using Ginger.Configurations;
+using Ginger.Run;
+using Ginger.SolutionGeneral;
+using Ginger.UserControls;
+using GingerCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Ginger.SolutionWindows
+{
+    /// <summary>
+    /// Interaction logic for AccessibilityRulePage.xaml
+    /// </summary>
+    public partial class AccessibilityRulePage : Page
+    {
+        Solution mSolution;
+        string AppName;
+        List<string> DefaultExcludeRulesList;
+        private AccessibilityConfiguration mAccessibilityConfiguration;
+        public AccessibilityRulePage()
+        {
+            InitializeComponent();
+            mSolution = WorkSpace.Instance.Solution;
+            string allProperties = string.Empty;
+            WorkSpace.Instance.PropertyChanged += WorkSpacePropertyChanged;
+            LoadGridData();
+            SetAppsGrid();
+            mAccessibilityConfiguration = WorkSpace.Instance.SolutionRepository.GetFirstRepositoryItem<AccessibilityConfiguration>();
+            mAccessibilityConfiguration.DefaultExcludeRule = mAccessibilityConfiguration.DefaultExcludeRule != null ? mAccessibilityConfiguration.DefaultExcludeRule : new ();
+        }
+        private void WorkSpacePropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(WorkSpace.Solution))
```

Comment:
As values are not being saved in solution xml, do we need this property change event?

Reply:
Done